consent-store/consent_store.py

```
@@ -3,6 +3,9 @@
 from database.sqlite_repository import SQLiteRepository
 from database.repository import DatabaseRepository
 from routers import applications, consent
+import sys
+sys.path.append('/app')  # Add app directory to path
+from config import *
```

Ruff (0.11.9)

`from config import *` used; unable to detect undefined names

(F403)